extensions/wikia/PhalanxII/tests/PhalanxModelTest.php

```diff
@@ -182,22 +182,22 @@ public function testPhalanxContentModelTitle( $title, $block, $result ) {
 	 * @dataProvider phalanxNewFromTypeProvider
 	 */
 	public function testPhalanxNewFromType( $type, $content, $className, $methodName ) {
-		$model = PhalanxModel::newFromType($type, $content);
+		$model = PhalanxModel::newFromType( $type, $content );
 
-		if ($className === false) {
-			$this->assertNull($model);
+		if ( $className === false ) {
+			$this->assertNull( $model );
 		}
 		else {
-			$this->assertInstanceOf($className, $model);
-			$this->assertEquals($content, $model->$methodName());
+			$this->assertInstanceOf( $className, $model );
+			$this->assertEquals( $content, $model->$methodName() );
 		}
 	}
 
 	public function phalanxNewFromTypeProvider() {
 		return array(
 			array(
 				'type' => Phalanx:: TYPE_TITLE,
-				'content' => $this->getMockWithMethods('Title', array('getText' => 'foo')),
+				'content' => $this->getMockWithMethods( 'Title', array( 'getText' => 'foo' ) ),
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "content" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)